src/django/api/serializers/v1/production_locations_serializer.py

```diff
@@ -45,6 +47,15 @@ class ProductionLocationsSerializer(Serializer):
         choices=['asc', 'desc'],
         required=False
     )
+    aggregation = ChoiceField(
+        choices=['hexgrid'],
```

Comment:
I would suggest saying `geohex_grid` instead of `hexgrid` to be more aligned how it's named in OpenSearch.

Reply:
Done.